.github/workflows/make_release.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+import argparse
+
+from packaging.version import Version
+import requests
+from uritemplate import expand as uri_expand
+
+
+def make_release(version, commit_hash, release_notes=""):
+    """Create a new GitHub release using the given data
+
+    This function always makes a pre-release first to ensure the "latest" release never corresponds
+    to one without artifacts uploaded. If the new version number is not a pre-release, as
+    determined by PEP-440, it is promoted to at full release after the uploads have completed
+    successfully.
+
+    :param str version: The version of the new release
+    :param str commit_hash: Git revision used for the release
+    :param str release_notes: Release notes
+    """
+    # Create a draft release
+    r = requests.post(
+        f"{api_root}/releases",
+        json={
+            "tag_name": version,
+            "target_commitish": commit_hash,
+            "body": release_notes,
+            "draft": True,
+            "prerelease": Version(version).is_prerelease,
+        },
+        headers=headers,
+    )
+    r.raise_for_status()
+    release_data = r.json()
+    print(f"Created draft release at: {release_data['html_url']}")
```

Comment:
This format is python 3 only, IIUC (f-strings).
And that is fine, of course, but in general, for our own sanity, can you use the old style that we are all used to?

Reply:
This applies in a few places in this file.

Reply:
This function is copied from DIRACOS2 and I don't want to start changing it's contents as it will just make maintenance a hassle if anything needs to be changed.

We also already have f-strings in [management](https://github.com/DIRACGrid/management/blob/master/dirac-distribution/WebAppCompiler.py) and the moment we can drop Python 2 we should starting using them in DIRAC. I know they're odd at first sight but you quickly become accustomed to them and they feel like the obvious and most readable way of formatting string. There are also technical benefits (e.g. much faster performance/better static analysis).